packages/react/src/test/htmlConversion.test.tsx

```diff
@@ -81,31 +77,32 @@ describe("Test React HTML conversion", () => {
   for (const testCase of testCases) {
     describe("Case: " + testCase.name, () => {
       let editor: BlockNoteEditor<any, any, any>;
-      let root: Root;
-      const div = document.createElement("div");
-
-      beforeEach(() => {
-        editor = testCase.createEditor();
-
-        const el = (
-          <TestContext.Provider value={true}>
-            <BlockNoteView editor={editor} />
-          </TestContext.Provider>
-        );
-        root = createRoot(div);
-        flushSync(() => {
-          // eslint-disable-next-line testing-library/no-render-in-setup
-          root.render(el);
-        });
-      });
-
-      afterEach(() => {
-        root.unmount();
-        editor._tiptapEditor.destroy();
-        editor = undefined as any;
-
-        delete (window as Window & { __TEST_OPTIONS?: any }).__TEST_OPTIONS;
-      });
+      // TODO: Why do we need to render for unit tests?
```

Comment:
to make sure the contexts such as TestContext are available (and an internal one that's necessary for `renderToDOMSpec`